tools/cosmovisor/cmd/cosmovisor/add_upgrade.go

```diff
@@ -93,7 +98,7 @@ func AddUpgrade(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		logger.Info(fmt.Sprintf("%s created, %s upgrade binary will switch at height %d", upgradetypes.UpgradeInfoFilename, upgradeName, upgradeHeight))
+		logger.Info(fmt.Sprintf("%s/%s created, %s upgrade binary will switch at height %d", cfg.UpgradeInfoFilePath(), upgradetypes.UpgradeInfoFilename, upgradeName, upgradeHeight))
```

Comment:
Maybe we should use `filepath.Join` for the 2 users using Windows Server :D

Reply:
If we stop using windows for all our servers, that can bring us down to just 1 user?